src/website/views/components/button-group.pug

```diff
@@ -115,6 +115,592 @@ p.-text
+          <div class="a-btn__content">
+            <div class="a-icon">
+              <svg><use xlink:href="#icon-atom"></use></svg>
+            </div>
+          </div>
+        </button>
+        <button class="a-btn -xl -icon">
+          <div class="a-btn__content">
+            <div class="a-icon">
+              <svg><use xlink:href="#icon-atom"></use></svg>
+            </div>
+          </div>
+        </button>
+      </div>
+      <div class="m-btnGroup">
+        <button class="a-btn -xl">
+          <div class="a-btn__content">
+            <div class="a-icon">
+              <svg><use xlink:href="#icon-atom"></use></svg>
+            </div>
+            <span>Button</span>
+          </div>
+        </button>
+        <button class="a-btn -xl">
+          <div class="a-btn__content">
+            <div class="a-icon">
+              <svg><use xlink:href="#icon-atom"></use></svg>
+            </div>
+          <span>Button</span>
+        </div>
+        </button>
+        <button class="a-btn -xl">
+          <div class="a-btn__content">
+            <div class="a-icon">
+              <svg><use xlink:href="#icon-atom"></use></svg>
+            </div>
+          <span>Button</span>
+        </div>
+        </button>
+      </div>
+
+  #vertical.a-tabs-panel
+    each size in ['sm', 'md', 'lg', 'xl']
+      .-d--flex.-flex--wrap
+        .m-btnGroup.-vertical.-fluid.-m--3(style="width:12rem")
+          button(class=`a-btn -${size}`) Button
+          button(class=`a-btn -${size}`) Button
+          button(class=`a-btn -${size}`) Button
+        .m-btnGroup.-vertical.-fluid.-pill.-m--3(style="width:12rem")
+          button(class=`a-btn -${size}`) Button
+          button(class=`a-btn -${size}`) Button
+          button(class=`a-btn -${size}`) Button
+        .m-btnGroup.-vertical.-fluid.-m--3
+          button(class=`a-btn -${size} -icon`)
+            .a-btn__content
+              .a-icon
+                svg
+                  use(xlink:href='#icon-atom')
+          button(class=`a-btn -${size} -icon`)
+            .a-btn__content
+              .a-icon
+                svg
+                  use(xlink:href='#icon-atom')
+          button(class=`a-btn -${size} -icon`)
+            .a-btn__content
+              .a-icon
+                svg
+                  use(xlink:href='#icon-atom')
+        .m-btnGroup.-vertical.-fluid.-m--3(style="width:12rem")
+          button(class=`a-btn -${size}`)
+            .a-btn__content
+              .a-icon
+                svg
+                  use(xlink:href='#icon-atom')
+              span Button
+          button(class=`a-btn -${size}`)
+            .a-btn__content
+              .a-icon
+                svg
+                  use(xlink:href='#icon-atom')
+              span Button
+          button(class=`a-btn -${size}`)
+            .a-btn__content
+              .a-icon
+                svg
+                  use(xlink:href='#icon-atom')
+              span Button
+      .-pl--3
```

Comment:
@davidojedacl can we move the size indicator to the beginning of each example instead of being at the bottom of each example?